toolkits/ob-is/repository/resources/wso2is-6.0.0-deployment-cds.toml

```diff
@@ -173,7 +173,6 @@ allowed_scopes = ["OB.*", "profile"]
 renew_refresh_token = false
 
 [oauth.endpoints]
-oauth2_token_url = "${carbon.protocol}://APIM_HOSTNAME:8243/token"
```

Comment:
This config should be there to resolve the audience value issue in PAR request

Reply:
Fixed